src/lib.rs

```diff
@@ -116,6 +122,15 @@ pub struct ParagraphInfo {
     pub level: Level,
 }
 
+
+
+impl ParagraphInfo {
+    /// Get the length of the paragraph in the source text.
+    pub fn get_length(&self) -> usize {
```

Comment:
nit: just `len()`

Reply:
done in PR: https://github.com/servo/unicode-bidi/pull/73